ecs-init/docker/docker_test.go

```diff
@@ -985,56 +971,6 @@ func TestDefaultIsPathValid(t *testing.T) {
 	}
 }
 
-func TestGetCredentialsFetcherSocketBind(t *testing.T) {
```

Comment:
Why are we deleting this test?

Reply:
The function this is testing gets the creds fetcher directory and then checks that it actually exists, so AFAICT it would only ever pass if run on an instance that has the creds fetcher installed: https://github.com/aws/amazon-ecs-agent/blob/61c8a8c5b48e2efb7193fe61c4bb3cae816f5821/ecs-init/docker/docker.go#L468

This seems outside the scope of unit tests to me so I have just deleted it.